oracle/terraform-provider-oci
# Update references to Oracle owned sdk repo

* this is still a private repo but there is no change to the contents of the respective vendor codefiles

### Test
* Verify successful compilation
* Execute fast running acceptance tests
